docs/fundamentals/code-analysis/overview.md

```diff
@@ -51,11 +59,25 @@ You can change the severity of these rules to disable them or elevate them to er
 
 *Analysis mode* refers to a predefined code analysis configuration where none, some, or all rules are enabled. In the default analysis mode, only a small number of rules are [enabled as build warnings](#enabled-rules). You can change the analysis mode for your project by setting the [\<AnalysisMode>](../../core/project-sdk/msbuild-props.md#analysismode) property in the project file. The allowable values are:
 
-| Value | Description |
-| - | - |
-| `AllDisabledByDefault` | This is the most conservative mode. All rules are disabled by default. You can selectively [opt into](configuration-options.md) individual rules to enable them.<br /><br />`<AnalysisMode>AllDisabledByDefault</AnalysisMode>` |
-| `AllEnabledByDefault` | This is the most aggressive mode. All rules are enabled as build warnings. You can selectively [opt out of](configuration-options.md) individual rules to disable them.<br /><br />`<AnalysisMode>AllEnabledByDefault</AnalysisMode>` |
-| `Default` | The default mode, where a handful of rules are enabled as warnings, others are enabled only as Visual Studio IDE suggestions with corresponding code fixes, and the rest are disabled completely. You can selectively [opt into or out of](configuration-options.md) individual rules to disable them.<br /><br />`<AnalysisMode>Default</AnalysisMode>` |
+:::row:::
+    :::column:::
+        `None`
+    :::column-end:::
+    :::column:::
+        `Default`
+    :::column-end:::
+    :::column:::
+        `Minimum`
+    :::column-end:::
+    :::column:::
+        `Recommended`
+    :::column-end:::
+    :::column:::
+        `All`
+    :::column-end:::
+:::row-end:::
```

Comment:
Nit: If there is no description provided for each of these, why wouldn't they just be a list?

Reply:
They could be a list - it just takes up less space to have them in columns rather than rows.